.github/workflows/format-pr.yml

```diff
@@ -0,0 +1,51 @@
+name: format-pr
```

Comment:
I would not add such an action, and in particular I don't think it is useful to run it as a cron job. If the check fails in a PR, one can just push a commit with formatting fixes; and if only PRs with correct format are merged, there's no need to check or create PRs on the master branch.